jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -132,6 +182,21 @@ def _validate_bunit(bunit, raise_error=True):
     return valid
 
 
+# ---- Functions that handle input from JWST FITS files -----
+
+def _jwst_all_to_glue_data(file_obj, data_label):
+    for hdu in file_obj:
+        if _validate_fits_image2d(hdu, raise_error=False):
+
+            ext = hdu.name.lower()
+            if ext in ('sci', 'asdf'):
```

Comment:
It had the reference to 'asdf' already. Since I'm not proficient in JWST jargon, I left it as is. But it looked weird to me. Now I removed the reference to 'asdf.